prov/gni/src/gnix_freelist.c

```diff
@@ -103,9 +103,7 @@ int _gnix_fl_init(int elem_size, int offset, int init_size,
 
 	int fill_size = init_size != 0 ? init_size : GNIX_FL_INIT_SIZE;
 
-	fl->refill_size = (refill_size != 0 ?
-			   refill_size :
-			   GNIX_FL_INIT_REFILL_SIZE);
+	fl->refill_size = refill_size;
```

Comment:
Remove GNIX_FL_INIT_REFILL_SIZE, since you are now requiring everyone to specify a refill size and there is no sentinel value that will use a default initial size.